### DISCUSSION TITLE
issues with hyprland (xwayland) the window supposed to be hidden isnt

### DISCUSSION THREAD

**Discussion by QAD6675** [winapps-org/winapps#146] [Support]
im using xfreerdp and i did exactly what the instructions said things run but the display is messy
![Screenshot_17-Jul_01-08-36_19459](https://github.com/user-attachments/assets/f21a1a9d-b457-48e7-b28e-1c725c7edf9a)
if i try to fulscreen the one in the back a bit of gets chopped off and the floating one is unresponsive
any help would be appreciated

**Comment by oskardotglobal** [comment]
App streaming is unsupported on wayland.
